extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/indexing/MSQTuningConfigTest.java

```diff
@@ -76,4 +77,39 @@ public void testEquals()
                   .usingGetClass()
                   .verify();
   }
+
+  @Test
+  public void testDefaultValuesForElements()
+  {
+    MSQTuningConfig msqTuningConfig = new MSQTuningConfig(null, null, null, null, null);
+    Assert.assertEquals(1, msqTuningConfig.getMaxNumWorkers());
+    Assert.assertEquals(100000, msqTuningConfig.getMaxRowsInMemory());
+    Assert.assertEquals(PartitionsSpec.DEFAULT_MAX_ROWS_PER_SEGMENT, msqTuningConfig.getRowsPerSegment());
+    Assert.assertEquals(null, msqTuningConfig.getMaxNumSegments());
+    Assert.assertEquals(IndexSpec.DEFAULT, msqTuningConfig.getIndexSpec());
+  }
+
+  @Test
+  public void testCustomValuesForElements()
+  {
+    MSQTuningConfig msqTuningConfig = new MSQTuningConfig(5, 200000, 5000, 10, IndexSpec.builder().build());
+    Assert.assertEquals(5, msqTuningConfig.getMaxNumWorkers());
+    Assert.assertEquals(200000, msqTuningConfig.getMaxRowsInMemory());
+    Assert.assertEquals(5000, msqTuningConfig.getRowsPerSegment());
+    Assert.assertEquals(Integer.valueOf(10), msqTuningConfig.getMaxNumSegments());
+    Assert.assertEquals(IndexSpec.builder().build(), msqTuningConfig.getIndexSpec());
+  }
+
+  @Test
+  public void testToString()
```

Comment:
Do we really need to test the `toString` method? AFAICT, the `toString` of this class is used only for debug logging.

Reply:
Sounds good, removing in the next commit.